### ISSUE TITLE
invoices for juggler

### ISSUE THREAD

**Issue by smadamit** [inbaz/ers#238]
Some jugglers will need an invoice. We have to clarify how we will handle this issue onsite.

**Comment by philip-n** [comment]
The "payment confirmed" mail currently states that an invoice will be attached. Does this affect this issue?

**Comment by smadamit** [comment]
@philip-n  Yes, that was the plan, but we have to discard it. We will not send invoices per email. So we have to change the email text.

**Comment by philip-n** [comment]
In the "payment received"-mail, change

	<p>Attached to this email, please find the following documents:
		<ul>
			<li>the invoice for your order</li>
			<li>your e-tickets</li>
		</ul></p>

to 

<p>Your e-tickets are attached to this email as pdf documents.</p>

**Comment by philip-n** [comment]
Sorry, hadn't realized that I was kidnapping tims issue. Reverted to original titel and assignee.

**Comment by inbaz** [comment]
Hi @jolieno,

welcome to the project. :) Here's your first issue. Since 2015 Bruneck was the host they were responsible for the invoices. Would be great if you could check who and how invoices are created and which data is needed to be able to do that.
